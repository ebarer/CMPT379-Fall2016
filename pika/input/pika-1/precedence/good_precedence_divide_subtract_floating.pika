exec {

  print 7.3 / 5.5 - 7.9, _n_.
  print 2.1 / 9.6 - 8.8, _n_.
  print 6.6 / 9.1 - 7.2, _n_.
  print 2.5 / 3.4 - 1.5, _n_.
  print 8.7 / 5.0 - 2.0, _n_.
  print 5.3 / 7.9 - 9.5, _n_.
  print 2.3 / 7.6 - 2.1, _n_.
  print 9.8 / 2.0 - 6.0, _n_.
  print 8.7 / 5.2 - 3.2, _n_.
  print 5.1 / 9.4 - 5.8, _n_.
  print 5.0 / 4.5 - 8.5, _n_.
  print 5.6 / 1.8 - 3.6, _n_.

  print 6.6 - 2.2 / 2.1, _n_.
  print 5.2 - 3.4 / 3.9, _n_.
  print 7.9 - 1.5 / 6.5, _n_.
  print 9.9 - 9.8 / 5.0, _n_.
  print 7.3 - 7.3 / 2.3, _n_.
  print 2.4 - 8.1 / 1.3, _n_.
  print 1.5 - 2.3 / 8.4, _n_.
  print 7.6 - 8.2 / 6.2, _n_.
  print 1.2 - 1.5 / 5.8, _n_.
  print 4.0 - 8.1 / 9.1, _n_.
  print 2.1 - 9.4 / 6.7, _n_.
  print 6.0 - 1.0 / 6.9, _n_.

  print 8.0 / 8.2 - 1.8 / 8.2, _n_.
  print 3.4 / 4.7 - 2.9 / 8.3, _n_.
  print 4.2 / 9.5 - 7.0 / 1.2, _n_.
  print 2.0 / 9.4 - 6.8 / 8.8, _n_.
  print 3.2 / 9.5 - 8.1 / 6.9, _n_.
  print 8.8 / 1.7 - 5.2 / 4.9, _n_.
  print 4.1 / 2.8 - 3.2 / 2.4, _n_.
  print 1.5 / 3.0 - 1.7 / 4.8, _n_.
  print 4.1 / 8.5 - 2.0 / 5.7, _n_.
  print 4.7 / 5.8 - 8.1 / 1.5, _n_.
  print 2.2 / 1.3 - 1.9 / 4.4, _n_.
  print 6.5 / 3.7 - 7.1 / 6.7, _n_.

  print 2.0 - 4.0 / 7.5 - 7.8, _n_.
  print 8.5 - 7.2 / 1.2 - 5.3, _n_.
  print 2.9 - 6.1 / 1.2 - 2.0, _n_.
  print 4.8 - 3.0 / 4.0 - 3.6, _n_.
  print 3.9 - 4.5 / 3.4 - 4.2, _n_.
  print 3.8 - 7.8 / 4.9 - 4.6, _n_.
  print 8.5 - 1.2 / 7.4 - 5.0, _n_.
  print 4.9 - 6.2 / 5.9 - 6.3, _n_.
  print 8.0 - 1.2 / 8.8 - 8.1, _n_.
  print 9.9 - 4.3 / 8.3 - 3.4, _n_.
  print 9.3 - 5.6 / 1.3 - 7.6, _n_.
  print 7.2 - 9.2 / 4.1 - 9.3, _n_.

}

