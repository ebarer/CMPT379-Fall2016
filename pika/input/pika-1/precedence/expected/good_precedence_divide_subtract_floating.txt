-6.57273
-8.58125
-6.47473
-0.764706
-0.26
-8.82911
-1.79737
-1.1
-1.52692
-5.25745
-7.38889
-0.488889
5.55238
4.32821
7.66923
7.94
4.12609
-3.83077
1.22619
6.27742
0.941379
3.10989
0.697015
5.85507
0.756098
0.374007
-5.39123
-0.559961
-0.837071
4.11525
0.130952
0.145833
0.131476
-4.58966
1.26049
0.697055
-6.33333
-2.8
-4.18333
0.45
-1.62353
-2.39184
3.33784
-2.45085
-0.236364
5.98193
-2.60769
-4.3439
